theme/templates/resource-landing-page/title-section.html

```diff
@@ -154,6 +154,41 @@
             </div>
         </div>
     {% endif %}
+    <div id="checking-for-paths" class="col-sm-12">
+        <p class="alert alert-warning">Checking for non-preferred file/folder path names (may take a long time depending on the number of files/folders) ...</p>
+    </div>
+    <div id="non-preferred-paths">
+        <div class="col-sm-12">
+            <div id="non-preferred-paths" class="alert {% if just_created %}alert-success{% else %}alert-warning{% endif %} alert-dismissible" role="alert">
```

Comment:
can a resource have non-preferred paths if it was `just_created`?

Reply:
No. Will fix that.